CHANGELOG

```diff
@@ -86,241 +173,6 @@
 - New Template for Cisco NX-OS: show forwarding adjacency [\#722](https://github.com/networktocode/ntc-templates/pull/722) ([Yakuza-UA](https://github.com/Yakuza-UA))
 - BugFix: cisco\_ios\_show\_interfaces\_switchport: Made trunking\_vlans a list, and changed regex [\#671](https://github.com/networktocode/ntc-templates/pull/671) ([FragmentedPacket](https://github.com/FragmentedPacket))
 
-## [1.4.2](https://github.com/networktocode/ntc-templates/tree/1.4.2) (2020-05-25)
-
-[Full Changelog](https://github.com/networktocode/ntc-templates/compare/v1.4.1...1.4.2)
```

Comment:
Why remove this?

Reply:
The changelog is autogenerated. I'm not sure that wasn't replaced before